lib/transport_web/controllers/page_controller.ex

```diff
@@ -1,8 +1,8 @@
 defmodule TransportWeb.PageController do
   use TransportWeb, :controller
 
-  def index(conn, _params) do
-    render conn, "index.html"
+  def index(conn, _) do
```

Comment:
You can let _params, it will be ignored, but we know what's in that unused variable

Reply:
Yep